### PR TITLE
Update on past ph page

### PR DESCRIPTION
**Pull request by Foukki**
Update PH UI

### CODE REVIEW

**Comment by maximeroucher** [comment]
Can you add a screen of the updated page

**Comment by Foukki** [comment]
Capture d’écran 2024-09-18 à 08 06 46

**Comment by Daihecyy** [comment]
Try reducing the size of the download button to avoid conflict between the rounded corners
The current UI suggests to me that you have to download the file to read it

**Comment by maximeroucher** [comment]
You can keep the title of the pdf, and add the "Télécharger" text alongside the icon on the button.
You can use some predefined component such as CardLayout and The ButtonCardLayout to keep consistency of the UI throughout the app

**Comment by Foukki** [comment]
Maybe title is not necessary because it is on the first page

**Comment by Foukki** [comment]
I suggest adding an option for radius in the CardLayout.

**Comment by Foukki** [comment]
Capture d’écran 2024-09-18 à 09 21 10

**Comment by Foukki** [comment]
I don't really get where you want the "Télécharger" button

File: lib/ph/ui/pages/past_ph_selection_page/ph_card.dart [hunk 1]
```diff
@@ -43,76 +42,69 @@ class PhCard extends HookConsumerWidget {
...
-                      color: Colors.black,
+
+                if (path != null) {
+                  displayPhToastWithContext(
+                    TypeMsg.msg,
+                    PhTextConstants.succesDowloading,
+                  );
+                }
+              },
+              child: Row(
```
Comment (julien4215) [inline lib/ph/ui/pages/past_ph_selection_page/ph_card.dart:91]: Do we need a row here ? There is only one children

**Comment by Foukki** [comment]
att 1Wrci6bWJyb0Nix5s4NhGN4_d0KMuc6UfzUJgN3BZhE

**Review by Daihecyy — APPROVED** [review]

**Review by mickaecl — APPROVED** [review]

**Pull request merged**